docs/concepts.mdx

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 2
+sidebar_label: "Concepts"
+id: concepts
+title: Kwil Concepts
+description: Core Concepts of Kwil
+slug: /concepts
+---
+
+# Kwil Concepts
+
+Because Kwil database [networks](#kwil-network) are sovereign blockchains, there are a few concepts that are unique or borrowed from other blockchain platforms. These concepts are transversal across Kwil (frontends, SDKs, Nodes, etc.).
+
+<div style={{ display: "flex", justifyContent: "center" }}>
+    <img src={require('../static/img/kwil-concepts.png').default} alt="Concepts Overview" />
+</div>
+
+## Accounts
+
+Accounts are public / private keypairs that represent a single user or entity on a Kwil Network. Accounts are used to deploy databases and execute CRUD operations.
+
+Each account has a public_key, nonce, and balance.
+
+Accounts are identified by public keys. Natively, Kwil supports Secp256k1 keys (used in EVM) and ED25519 keys (used in Solana, NEAR, etc).
+
+## Databases 
+
+In Kwil, databases refer to SQL deployments on a Kwil Network. The Kwil [SQL engine](/docs/category/sql-as-understood-by-kwil) is based on SQLite with modifications to support deterministic execution and consensus.
+
+### Kuneiform
+
+[Kuneiform](/docs/category/kuneiform) is the syntax language for defining a database on Kwil. You can think of Kuneiform as a smart contract language (e.g. [Solidity](https://soliditylang.org/)), but for relational databases.
+
+In a Kuneiform file, you can define:
+- Relational Tables
+- Actions - CRUD Operations that can be executed on the database.
+- Extensions - Custom logic that can be executed on the database.
+
+### Database Identifiers
+
+Database identifiers (DBIDs) are unique identifiers for a database on a [Kwil Network](#kwil-network). DBIDs are generated from the public key of the account that deployed the database and the database name. A DBID is a SHA224 hash of the database owner's account public key and database name.
```

Comment:
I guess this is nbd, the hash is `hash(name+public_key)`, so maybe change to `hash of database name and ...key`